client/.env.example

```diff
@@ -3,3 +3,4 @@
 #
 # default. own preference might be used
 REACT_APP_SOCKET_SERVER=http://localhost:3002
+REACT_APP_API_PATH=/api
```

Comment:
`api prefix` maybe?

Reply:
I named it as in the server. I just called the task before I looked at it

Reply:
Got it